locale/templates/LC_MESSAGES/server.pot

```diff
@@ -665,6 +665,56 @@ msgid ""
 "automatically renew each billing period unless you choose to cancel."
 msgstr ""
 
+#: lib/senders/templates/subscriptionRenewalReminder.html:1
+#: lib/senders/templates/subscriptionRenewalReminder.txt:1
+msgid "Your subscription will be renewed soon"
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.html:2
+#: lib/senders/templates/subscriptionRenewalReminder.txt:2
+msgid ""
+"Your current subscription is set to automatically renew in "
+"%(reminderLength)s days. At that time, Mozilla will renew your "
+"%(planIntervalCount)s %(planInterval)s subscription and a charge of "
+"%(invoiceTotal)s will be applied to the payment method on your account."
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.html:3
+msgid ""
+"You can ensure that your payment method and account information are up to "
+"date <a href=\"%(updateBillingUrl)s\">here</a>."
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.html:4
+msgid ""
+"Thank you for subscribing to %(productName)s. If you have any questions "
+"about your subscription or need more information about %(productName)s "
+"please <a href=\"%(subscriptionSupportUrl)s\">contact us</a>."
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.html:5
+#: lib/senders/templates/subscriptionRenewalReminder.txt:5
+msgid "Sincerely,"
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.html:6
+#: lib/senders/templates/subscriptionRenewalReminder.txt:6
+msgid "The %(productName)s team"
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.txt:3
+msgid ""
+"You can ensure that your payment method and account information are up to "
+"date here:"
+msgstr ""
+
+#: lib/senders/templates/subscriptionRenewalReminder.txt:4
+msgid ""
+"Thank you for subscribing to %(productName)s. If you have any questions "
```

Comment:
Need to have a comma **before** `please contact us.`

Reply:
done